components/CreateModeSelect/CreateModeSelect.tsx

```
@@ -1,55 +1,22 @@
-            >
-              new link
-            </CTAButton>
-            <CTAButton
-              isActive={isEmbedPage}
-              onClick={() => push(`/create/embed${urlQuery}`)}
-            >
-              new embed
-            </CTAButton>
-          </div>
-        </div>
+        <Fragment>{isMobile ? <MobileSelect /> : <DesktopSelect />}</Fragment>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)